nodemon.json

```diff
@@ -0,0 +1,10 @@
+{
+  "ext": "js,json",
```

Comment:
If removing nodemon, we should remove this

Reply:
For sure, I'll pull it out in the branch where I'm trying to get tests to run in CI.